modules/swagger-parser-core/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>io.swagger.parser.v3</groupId>
         <artifactId>swagger-parser-project</artifactId>
-        <version>2.0.12</version>
```

Comment:
We need to change the groupId to com.github.krraghavan (else we can't deploy it).  Change the version to 2.0.12.1

Reply:
fixed